package/setup.py

```diff
@@ -58,5 +58,7 @@
             "sphinx_rtd_theme==1.3.0",
             "myst-parser>=1.0,<2.1",
         ],
+        "aws": ["s3fs>=2021.4,<2024.1"],
+        "azure": ["adlfs>=2021.4,<2024.1"],
```

Comment:
Is there any documentation explains why this pin differently from `kedro`? It would be nice even if it's just an in-line comment. Some users were asking why this is pinned, from my understanding it only affect kedro-viz user who is using the static site deployment.

Reply:
sure, the issue was files were being copied differently from `fsspec>=2024.1`. The project structure was not maintained. But, you are right I will try to see if we can remove the upperbound, if not will document this with a comment here and also in the deployment docs. Thank you !

Reply:
Hi @noklam , I tested with the latest version of fsspec, s3fs and adlfs. The deployment seems to work fine. I would request @jitu5 to test it and confirm. If it works fine, we will drop the upperbound and sync it with kedro. Thanks for pointing this 👍

Reply:
@ravi-kumar-pilla  I tested with the latest version of fsspec, s3fs and adlfs and its works fine for me.